spa/src/components/molecules/CookiesBox.vue

```diff
@@ -70,13 +70,64 @@ export default {
   },
   data() {
     return {
-      checked: false,
+      statistical: false,
+      marketing: false,
+      preferencesChosen: false,
     };
   },
+  methods: {
+    confirmAll() {
+      this.statistical = true;
+      this.marketing = true;
+      this.savePreferences();
+    },
+    savePreferences() {
+      const confirmDate = new Date();
+      const options = { statistical: this.statistical, marketing: this.marketing, confirmDate };
+
+      localStorage.setItem('cookieSettings', JSON.stringify(options));
+
+      this.emitEvent(options);
+      this.preferencesChosen = true;
+    },
+    emitEvent(options) {
+      this.$emit('cookiePreferenceChange', options);
+    },
+    loadCookiePreferences() {
+      let options;
+      try {
+        options = JSON.parse(localStorage.getItem('cookieSettings'));
+
+        if (!options || !options.confirmDate) {
+          this.resetCookiePreferences();
+          return;
+        }
+        this.statistical = options.statistical;
+        this.marketing = options.marketing;
+        this.preferencesChosen = true;
+        this.emitEvent(options);
+      } catch (err) {
+        this.resetCookiePreferences();
+      }
+    },
+    resetCookiePreferences() {
+      this.statistical = false;
+      this.marketing = false;
+      this.preferencesChosen = false;
+
+      const options = { statistical: false, marketing: false, confirmDate: null };
+      localStorage.setItem('cookieSettings', JSON.stringify(options));
```

Comment:
and here you can use the first vuex store method to save cookies into local storage